scripts/run-on-error.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+const execa = require('execa')
+const { argv } = require('process')
+
+const [, , npmScript, npmScriptOnError] = argv
```

Comment:
This script is only intended for our npm scripts, so I am not sure whether using a CLI input parsing library is needed.